cubes/query/browser.py

```diff
@@ -75,6 +77,41 @@
 _ReportResult = Union[AggregationResult, Facts, JSONType, List[JSONType]] 
 
 
+class BrowserFeatureAction(Enum):
+    aggregate = 1
+    fact = 2
+    facts = 3
+    members = 4
+    cell = 5
+
+    @classmethod
+    def from_string(cls, actions: Iterable[str]) -> List['BrowserFeatureAction']:
```

Comment:
It is not clear from the function name that the return value is a list. By name I would expect it to return one value of itself, which is the same as `BrowserFeatureAction[astring]`. If we want to return a list of them, I would name it more verbosely `actions_from_strings`, if this is needed at all.